src/atwho.js

```diff
@@ -17,20 +17,21 @@ define([
 
     // stripped down version of http://codepen.io/ImagineProgramming/storydump/javascript-memoization-timeout
     var timed = function timed(f, timeout) {
-        var cache, time = 0;
+        var cache = {}, time = 0;
```

Comment:
If there is a mug in the place where you need to reference it, you can do `mug.form.vellum.data.atwho`